### PR TITLE
Conferences - add Forum PHP Paris 2024

### PR DESCRIPTION
**Pull request by ternel**

### CODE REVIEW

File: archive/entries/2024-06-10-1.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>Forum PHP 2024 - Paris (France)</title>
+  <id>https://www.php.net/archive/2024.php#2024-06-07-1</id>
```
Comment (saundefined) [inline archive/entries/2024-06-10-1.xml:4]:
```suggestion
  <id>https://www.php.net/archive/2024.php#2024-06-10-1</id>
```

**Review by saundefined — COMMENTED** [review]
Thanks!

It seems in one place the identifier should be fixed, otherwise LGTM

**Comment by ternel** [comment]
Thanks, good catch :)

**Review by saundefined — APPROVED** [review]
Thanks!

**Pull request merged**